go.work

```diff
@@ -1,4 +1,4 @@
-go 1.22.7
+go 1.24.0
```

Comment:
In theory if we remove the toolchain in the `go.mod` this should just be `1.24` 😄

Reply:
Yes, removed `.0` here:
[`c5d0d67` (#9)](https://github.com/noble-assets/jester/pull/9/commits/c5d0d6702a546791fc9c8d89be5318a2da9b97cc)